build/android/app/src/main/java/net/minetest/minetest/UnzipService.java

```diff
@@ -106,7 +106,7 @@ private void unzip(Intent intent) {
 		int readLen;
 		byte[] readBuffer = new byte[8192];
 		try (FileInputStream fileInputStream = new FileInputStream(zipFile);
-		     ZipInputStream zipInputStream = new ZipInputStream(fileInputStream)) {
+			 ZipInputStream zipInputStream = new ZipInputStream(fileInputStream)) {
```

Comment:
one space too much

Reply:
Interesting. I haven't touched it. Moreover, this piece of code does not have extra space in my Android Studio project. Would Github reformat code automatically?

Reply:
Some text editors suck. They merge spaces which follow on tabulators into the tabulator so that you cannot select and remove it. Navigate to the first `Z`, clear all indents and re-indent.